### ISSUE TITLE
Commit 0f34844009075391c55419a3afb90e469facf35e broke Windows builds

### ISSUE THREAD

**Issue by gohrner** [GrahamDumpleton/mod_wsgi#437]
The following commit:

https://github.com/GrahamDumpleton/mod_wsgi/commit/0f34844009075391c55419a3afb90e469facf35e#diff-eeae46a441a6bf04051e30249e5440bd

seems to have broken Windows builds.

posixmodule.c in Python has a legacy wrapper (noop) in case of windows for PyOS_AfterFork() but not for PyOS_AfterFork_Child():

```
/*` Legacy wrapper */
void
PyOS_AfterFork(void)
{
#ifdef HAVE_FORK
PyOS_AfterFork_Child();
#endif
}
```

So in case of HAVE_FORK = 0 (windows) no method PyOS_AfterFork_Child is available and thus the linking fails.

```
C:\> pip install mod_wsgi

(...)

mod_wsgi.obj : error LNK2001: unresolved external symbol PyOS_AfterFork_Child
build\lib.win-amd64-3.7\mod_wsgi\server\mod_wsgi.cp37-win_amd64.pyd : fatal error LNK1120: 1 unresolved externals
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\link.exe' failed with exit status 1120
```

On the contrary, `pip install mod_wsgi==4.6.4` (which does not contain this commit) works fine.

**Comment by GrahamDumpleton** [comment]
I am aware of the issue, see https://github.com/GrahamDumpleton/mod_wsgi/issues/436

You can also install 4.6.5 as was only introduced recently in 4.6.6. Am currently busy on trip and need to find some time to fix it.

**Comment by GrahamDumpleton** [comment]
Fixed in 4.6.7.
